.github/workflows/publish-python-wrapper-client.yaml

```diff
@@ -32,4 +32,37 @@ jobs:
         packages-dir: clients/python-wrapper/dist/
         verbose: true
 
-# TODO: Publish docs once we decide on documentation process
+    - name: Extract docs version
+      working-directory: ./clients/python-wrapper/
+      shell: bash
+      run: |
+          echo "tag=$(python setup.py --version)" >> $GITHUB_OUTPUT
+      id: docver
+
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.9'
+        cache: 'pip'
+        cache-dependency-path: 'clients/python-wrapper/requirements.txt'
+      run:
+        pip install -r requirements.txt
+    
+    - name: Generate and Build Documentation
+      working-directory: ./clients/python-wrapper/
+      run: |
+        sphinx-apidoc -o docs lakefs sphinx-apidoc --full -A 'Treeverse' -e
+        sphinx-build -b html docs docs/_build/
+        sphinx-build -b html docs docs/_build/${{ steps.docver.outputs.tag }}
```

Comment:
We require to build the generated files with different base url - it will probably need a different way to support this feature.

Reply:
The command builds it with relative references